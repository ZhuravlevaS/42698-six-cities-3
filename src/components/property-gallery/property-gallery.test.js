import React from 'react';
import PropertyGallery from './property-gallery.jsx';
import renderer from 'react-test-renderer';
import images from '../../mocks/imgs.js';

it(`<PropertyGallery/> render`, () => {
  const tree = renderer
    .create(<PropertyGallery images={images} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
