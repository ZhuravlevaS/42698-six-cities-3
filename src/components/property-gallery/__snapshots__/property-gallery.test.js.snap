// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PropertyGallery/> render 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/apartment-01.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/room.jpg"
      />
    </div>
  </div>
</div>
`;
